packages/pwa-kit-create-app/scripts/trim-extensions.js

```diff
@@ -66,9 +76,17 @@ function processFile(filePath, plugins) {
     const source = fs.readFileSync(filePath, 'utf-8')
 
     // Search file content for plugin references
-    const pluginRegex = new RegExp(Object.keys(plugins).join('|'))
-    const hasPluginReferences = pluginRegex.test(source)
-    if (!hasPluginReferences) {
+    const pluginPositions = []
+    const pluginRegex = new RegExp(Object.keys(plugins).join('|'), 'g')
+    let match
+    while ((match = pluginRegex.exec(source)) !== null) {
+        pluginPositions.push({
+            start: match.index,
+            end: match.index + match[0].length,
+            name: match[0] // store which plugin was found
+        })
+    }
+    if (pluginPositions.length === 0) {
```

Comment:
Updated the regex check so we only do it once per file instead of once per node. Thanks @bfeister for the good suggestion.